NeoML/Python/neoml/Dnn/Dnn.py

```diff
@@ -37,75 +47,100 @@ def __init__(self, math_engine, random=None):
 
     def store(self, path):
-        """
+        """Serializes the network.
+        
+        Parameters
+        ---------
+        path : str
+            The full path to the location where the network should be stored.
         """
         self._store(str(path))
 
     def load(self, path):
-        """
+        """Loads the network from file.
+        
+        Parameters
+        ---------
+        path : str
+            The full path to the location from where the network should be loaded.
         """
         self._load(str(path))
 
     def store_checkpoint(self, path):
-        """
+        """Serializes the network with the data required to resume training.
+        
+        Parameters
+        ---------
+        path : str
+            The full path to the location where the network should be stored.
         """
         self._store_checkpoint(str(path))
 
     def load_checkpoint(self, path):
-        """
+        """Loads the checkpoint from file.
+        A new solver will be created, because the old pointers will point
+        to an object no longer used by this network.
+        
+        Parameters
+        ---------
+        path : str
+            The full path to the location from where the network should be loaded.
         """
         self._load_checkpoint(str(path))
 
     @property
     def math_engine(self):
-        """
+        """Gets the math engine used by the network.
         """
         return self.get_math_engine()
 
     @property
     def solver(self):
-        """
+        """Gets the optimizer for the layer's trainable parameters.
         """
         return self.get_solver()
 
     @solver.setter
     def solver(self, new_solver):
-        """
+        """Sets the optimizer for the layer's trainable parameters.
         """
         self.set_solver(new_solver._internal)
 
     @property
     def initializer(self):
-        """
+        """Gets the initializer that will fill in the weight values 
+        before training starts. Xavier initialization is the default.
         """
         return self.get_initializer()
 
     @initializer.setter
     def initializer(self, new_initializer):
-        """
+        """Sets the initializer that will fill in the weight values 
+        before training starts.
         """
         self.set_initializer(new_initializer._internal)
 
     @property
     def input_layers(self):
-        """
+        """Gets the source layers of the network.
         """
         return self.get_inputs()
 
     @property
     def output_layers(self):
-        """
+        """Gets the sink layers of the network.
         """
         return self.get_outputs()
 
     @property
     def layers(self):
-        """
+        """Gets all layers of the network.
         """
         return self.get_layers()
 
     def run(self, inputs):
         """Runs the network.
+        
         Parameters
         ----------
         inputs : The dictionary of input blobs.
```

Comment:
It's better to clarify here, what dictionary must contain.

keys in dict: source layer names (type: str)
values in dict: blobs, which will be given to the corresponding sources (type: neoml.Blob)